src/kata5.reachDestination.js

```diff
@@ -1,3 +1,7 @@
-const reachDestination = (distance, speed) => {};
+const reachDestination = (distance, speed) => {
+  const time = distance / speed;
+  const timeRounded = Math.round(time * 2) / 2;
```

Comment:
This is perfectly working here, and no need to change, I would personally not assign the distance / speed to a variable, just add it directly into the math.round, (instead of time), as it is not being used anywhere else only once there.